README.md

```diff
@@ -108,9 +108,9 @@ If you have any problems building, please consult the developer [FAQ][].
 
 ### Build status of nightly builds
 
-| AppVeyor (Windows)       | Travis CI (Linux / macOS) | Code Coverage Status |
-|--------------------------|---------------------------|----------------------|
-| [![av-nightly-image][]][av-nightly-site] | [![tv-nightly-image][]][tv-site] | [![cc-image][]][cc-site] |
+| AppVeyor (Windows)       | Travis CI (Linux) | Travis CI (OSX)  | Code Coverage Status |
```

Comment:
I believe Apple has standardized on macOS vs OSX, so we should be consistent with macOS (ignore the fact we have $isOSX, unfortunately it's a breaking change now)

Reply:
shouldn't we be rather self consistent? I believe that it will be more confusing for our users unless we also make the change in our variables

Reply:
Agree we should be consistent with the variables, but that's outside the scope of this PR.  I think at least within the README.md, we can be consistent on using macOS (which it is currently)

Reply:
@SteveL-MSFT updated to include this change

Reply:
Still shows OSX, did you push?

Reply:
@SteveL-MSFT yup